src/views/EventPage.vue

```diff
@@ -0,0 +1,266 @@
+        <p v-else>{{ event.pub.description }}</p>
+        <h5 v-if="snipped" @click="snipped = false">Read More</h5>
+        <h5 v-else @click="snipped = true">See Less</h5>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="js">
+import NavBtns from "../components/navButtons";
+import ExploreBar from "../components/exploreBar";
+
+export default {
+  components: {
+    NavBtns,
+  },
+  filters: {
+    snippet(value) {
+      if (value.length > 150) {
+        return value.slice(0, 150) + "...";
+      } else {
+        return value;
+      }
+    },
+    hardSnippet(value) {
+      if (value.length > 300) {
+        return value.slice(0, 300) + "...";
+      } else {
+        return value;
+      }
+    }
+  },
+  props: {
+    event: Object,
+  },
+  data() {
+    return {
+      // navigation component options
+      navOptions: {
+        topType: 'share-light',
+        backRoute: '/discover'
+      },
+      snipped: true
+    };
+  },
+  computed: {
+    activeTags() {
+      const tagList = this.event.eventTags.host.filter(tag => {
+        return tag !== '';
+      });
+      if (tagList.length > 3) {
+        return tagList.slice(0, 3);
+      } else {
+        return tagList;
+      }
+    },
+    activeImages() {
+      const imageList = this.event.mediaLink.host;
+      if (imageList.length > 3) {
+        return imageList.slice(0, 3);
+      } else {
+        return imageList;
+      }
+    }
+  },
+  methods: {
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+/* DESKTOP */
+@media screen and (min-width: 1000px) {
+  .event-container {
+    max-width: 30%;
+  }
+  html {
+    border-style: none;
+    overflow-x: hidden;
+  }
+  img {
+    border-style: none;
+  }
+}
+
+/* MAIN DIV */
+.page-container {
+  @include page-container-middle--scrollable;
+
+  .nav-btns {
+    z-index: 1000;
+  }
+
+  /* BACKGROUND IMG */
+  .background-img {
+    @include background-img;
+
+    .gradient-overlay {
+      @include background-img-gradient(true);
+    }
+  }
+
+  /* ALL EVENT DETAILS */
+  .event-container {
+    position: absolute;
+    top: 30%;
+    z-index: 10;
+    width: 85%;
+    height: auto;
+    margin-bottom: 10vh;
+    // max-height: 70vh;
+
+    /* IMAGE GALLERY */
+    .gallery {
+      width: 100%;
+      height: 10%;
+      display: flex;
+      justify-content: flex-end;
+      flex-direction: row;
+      padding-bottom: 5%;
+
+      .image-container {
+        height: 4em;
+        width: 4em;
+        margin: 0 0.5em;
+        background-color: $primary-alt;
+        border-radius: 10px;
+        @include flex-middle;
+
+        img {
+          object-fit: fill;
+          height: 100%;
+          width: 100%;
+          border-radius: 10px;
+        }
+
+        h4 {
+          font-size: 1.5em;
+          font-weight: 600;
+          font-family: $text-primary;
+          color: white;
+        }
+      }
+    }
+
+    /* EVENT TAGS */
+    .event-tags {
```

Comment:
<img width="398" alt="Screen Shot 2020-03-10 at 12 11 10 AM" src="https://user-images.githubusercontent.com/44561376/76288791-ce8fbe80-6264-11ea-897e-cc41eeeff250.png">
"tag overflow" (when tags fill up larger than div width) causes overflow and x-scroll. I think could both disable x-scrolling and make the `event-tags` container wrap on overflow

Reply:
i'm gonna disable x scrolling and snip the length of the text to 8 chars

Reply:
Snip the length of the text?? :o

Reply:
yeaa like with ... at the end
or can we like just do x scrolling for the tags ya with no scrollbar tho

Reply:
Oh scrolling for tags is actually a good idea!! I wouldn't do the snipping tho, don't we want the tags to be readable?